MinimOsd_Extra/protocols/NMEA.h

```diff
@@ -316,14 +300,15 @@ store to msg.nmea.lat
 
 
 void read_NMEA(){
+    char string[NMEA_BUF_LENGTH];
     while (Serial.available_S()) {
```

Comment:
I am wondering now, did you ever test NMEA.h and changes you made before announce it?